CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Interaction responses are now passed the respective `proxy` and `proxy_auth` params as defined in `Client`.
   ([#1655](https://github.com/Pycord-Development/pycord/pull/1655))
 
+### Changed
+- Rename `original_message` to `original_response` ([#1609](https://github.com/Pycord-Development/pycord/pull/1609)
+
```

Comment:
```suggestion
```
Please add `original_message` to the deprecated section and `original_response` to the added section.

Reply:
Should I add separate entries for `original_message`, `edit_original_message` and `delete_original_message`?

Reply:
> Should I add separate entries for `original_message`, `edit_original_message` and `delete_original_message`?

no, but reference them all in the same entry